assemblies/collaborating-on-workbenches-in-code-server-by-using-git.adoc

```diff
@@ -0,0 +1,21 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+[id="collaborating-on-workbenches-in-code-server-by-using-git_{context}"]
+= Collaborating on workbenches in code-server by using Git
+
```

Comment:
+1, depending on how much forward looking we want to be, when writing the table, these could be taken into account

* the workbench/notebook naming consistency tickets, https://issues.redhat.com/browse/RHOAIENG-1099 
  * all the other tickets linked to this one, 
* request for a glossary, https://issues.redhat.com/browse/RHOAIENG-6605
* requesting a list of approved naming choices, https://issues.redhat.com/browse/RHOAIENG-10605

Furthermore, there is a request for remote IDEs and there's the Notebooks 2.0 rebase coming late in 2025; while the implications of the latter are still unclear. the former may require some doc rewrites, so it might be reasonable to plan for it ahead and be sufficiently clear in terminology to minimize required rewrites?

* the remote IDE rfe, https://issues.redhat.com/browse/RHOAISTRAT-320, also quite unclear, not sure if it's even possible to future-proof oneself for this

Reply:
Will incorporate these comments wtih RHOAI-4435